README.md

```diff
@@ -1,25 +1,24 @@
 # DemPoRT
-**Dementia Population Risk Tool (DemPoRT)**
 
-The Dementia Population Risk Tool (DemPoRT) is a predictive algorithm for calculating 5-year dementia risk, designed specifically for population-level use.  DemPoRT is discriminating and well-calibrated across a wide range of population subgroups, despite using only self-reported risk factors (with a focus on health behaviours including smoking, diet and physical inactivity).
+**Dementia Population Risk Tool (DemPoRT)**
 
-Most dementia algorithms are unsuitable for population-level assessment and planning as they are desinged for use in the clinical setting. DemPoRT is the first dementia risk algorithm designed for population use and can be used to produce improved estimates of current and future dementia burden, identify high risk population subgroups, and inform the development of dementia prevention strategies. It can also be used by patients and their clinicians to assess individual dementia risk.
+The Dementia Population Risk Tool (DemPoRT) is a predictive algorithm for calculating 5-year risk for developing dementia. DemPoRT was developed population-level use and it is discriminating and well-calibrated across a wide range of population subgroups, despite using only self-reported risk factors (with a focus on health behaviours including smoking, diet and physical inactivity). DemPoRT uses include predicting how many people will develop dementia, calculate future dementia burden, identify high risk population subgroups, and inform the development of dementia prevention strategies. It can also be used by patients and their clinicians to assess an individual's risk of developing dementia.
 
 <ins>Stay tuned for individual and population-level DemPoRT implementation tools at www.projectbiglife.ca</ins>
 
 DemPoRT was developed and validated using Ontario individuals age 55+ from the 2001 to 2012 Canadian Community Health Surveys (CCHS). There were 75,460 survey respondents, 8,448 dementia events, and 348,677 person-years follow-up in the combined derivation and validation cohorts.
```

Comment:
I a licence wouldn't normally include this level (performance of the application). Suggest removing but would reference the CMAJ paper.